ansible/roles/test/files/ptftests/advanced-reboot.py

```diff
@@ -125,6 +125,7 @@ def __init__(self):
         self.sad_handle = None
         self.process_id = str(os.getpid())
         self.test_params = testutils.test_params_get()
+        self.check_param('kvm_test', False, required=False)
```

Comment:
Can you also please update the script documentation with the new support that is being added?

Reply:
Not sure if the script means the comment lines at the beginning of the file. Added a line there showing the new support.

Reply:
If it is easy to detect that the DUT is KVM, we don't need add this new parameter.

---
In reply to: [512448831](https://github.com/Azure/sonic-mgmt/pull/2392#discussion_r512448831) [](ancestors = 512448831)